index.html

```diff
@@ -290,5 +290,60 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+
+    <!-- Start of the contact form -->
+    <footer class="form-body">
+      <div class="card-contact">
+        <h2 class="contact-text">Contact me</h2>
+
+        <p class="contact-para">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I`d love to help
+          with it
+        </p>
+
+        <form action="https://formspree.io/f/mpzbdpra" method="POST">
+          <ul class="form-content">
+            <li>
+              <label for="name"></label>
+              <input
+                class="form-list"
+                id="name"
+                type="text"
+                name="user_name"
+                placeholder="First Name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <label for="mail"></label>
+              <input
+                class="form-list1"
+                id="mail"
+                type="email"
+                name="user_email"
+                placeholder="Email ID"
+                required
+              />
+            </li>
+            <li>
+              <label for="msg"></label>
+              <textarea
+                class="form-list-textarea"
+                id="msg"
+                name="user_message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li class="button-list">
+              <button class="button1" type="submit">Get It Touch</button>
+            </li>
+          </ul>
```

Comment:
- [ ] Kindly use flex properties to layout the elements.
- [ ] Kindly add some button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

Reply:
![review1](https://user-images.githubusercontent.com/52527576/165321758-a96ad05f-4c02-4fbf-8a17-bb33f55c72d3.png)
I have used flexbox for the layout.
![hover](https://user-images.githubusercontent.com/52527576/165322004-9fd2b262-3490-48ac-9bb9-54a20ec2db0e.png)
I have used hover also. It is just that during inspect it is not visible.